packages/terra-consumer-nav/src/components/nav-burger-button/NavBurgerButton.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../variables';
+
+:local {
+  .burger {
+    background: var(--terra-consumer-burger-background, none);
+    border: var(--terra-consumer-burger-border, 0);
+    box-shadow: 0;
+    color: var(--terra-consumer-burger-color, #fff);
+    display: block;
+    font-size: 20px;
+    padding: 0;
+    position: absolute;
```

Comment:
These shouldn't be part of the toggler but rather the component that wraps it (which would be part of layout).